src/main/java/mate/academy/rickandmorty/repository/CharacterRepository.java

```diff
@@ -0,0 +1,14 @@
+package mate.academy.rickandmorty.repository;
+
+import java.util.List;
+import mate.academy.rickandmorty.model.SeriesCharacter;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.JpaSpecificationExecutor;
+import org.springframework.data.repository.PagingAndSortingRepository;
+
+public interface CharacterRepository extends JpaRepository<SeriesCharacter, Long>,
+        PagingAndSortingRepository<SeriesCharacter, Long>,
+        JpaSpecificationExecutor<SeriesCharacter> {

```

Comment:
JPA Repository already has the PagingAndSortingRepository.
```suggestion
public interface CharacterRepository extends JpaRepository<SeriesCharacter, Long>,
        JpaSpecificationExecutor<SeriesCharacter> {

```